sgp4/propagation.py

```diff
@@ -1335,8 +1335,8 @@ def sgp4init(
      #  ------------------------ earth constants -----------------------
      #  sgp4fix identify constants and allow alternate values
      #  this is now the only call for the constants
-     (satrec.tumin, satrec.mu, satrec.radiusearthkm, satrec.xke,
-       satrec.j2, satrec.j3, satrec.j4, satrec.j3oj2) = whichconst;
+     (satrec.tumin, satrec.mu, satrec.radiusearthkm, satrec.xke, 
+       satrec.j2, satrec.j3, satrec.j4, satrec.j3oj2) = getgravconst('wgs72');
```

Comment:
This looks like it breaks the previous API we still need to support, which used the gravity parameters passed in. Here you are still accepting the parameter but not using it?

Reply:
This might need to be re-written.  satrec.whichconst doesn't exist in the new satellite object.  We could accept it, then throw it out.  It's part of the larger problem in the new object of hard-coding wgs72 which is highlighted in issue https://github.com/brandon-rhodes/python-sgp4/issues/47

In this case, the backward compatibility appears to be compromised.

Reply:
Could you show the exception that was raised when this class was left as-is? I'm not sure I've yet caught the problem you're trying to solve through this change.

Reply:
Reverted to original code, with recent updates to whichconst.